configuration/filters.md

```diff
@@ -0,0 +1,90 @@
+---
+description: >-
+  This document outlines the various Filters types supported by the Edge Delta
+  agent, and how the filters are configured for inputs.
+---
+
+# Filters
+
+## Overview
+
+Filters are the mechanism to filter and transform the received logs from input sources.
+
+Proper filter configuration reduces resource usage, increases performance and hides or masks confidential data. It also provides a way to group the related events before further processing.
+
+Filters apply right after the data ingestion from input and before any processing is done in the workflows associated with the input.
+
+Filters should be defined at the top level, under filters section:
+
+```go
+  filters:
+    - name: error
+      type: regex
+      pattern: "error"
+```
+Filter names are used to apply filters to inputs.
+
+Below is an example file input with "error" and "mask_card" filters:
```

Comment:
also mention they are applied in given order

Reply:
Done